### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by HolyWu** [HolyWu/vs-rife#9] [Announcements]
- Only VS API4 is supported now.
- Update to 4.0 model and remove all old models.

<hr /><em>This discussion was created from the release <a href='https://github.com/HolyWu/vs-rife/releases/tag/v2.0.0'>v2.0.0</a>.</em>

**Comment by hzwer** [comment]
Amazing work!

**Comment by pdr0github** [comment]
Thanks

What was the reason for removing the old models ?  

There are situations where a different model can produce slightly better (or worse) results

**Reply by HolyWu** [reply]
Because keeping a pile of old versions is burdensome to me. Follow https://github.com/hzwer/Practical-RIFE#report-bad-cases if you really care and want it improved.

**Reply by pdr0github** [reply]
ok thanks 

Please leave the old versions archived (I can call different versions in the same script by renaming the rife folder and using different namespaces)
